legal-api/src/legal_api/models/business.py

```diff
@@ -126,6 +126,15 @@ def delivery_address(self):
         return db.session.query(Address).filter(Address.business_id == self.id).\
             filter(Address.address_type == Address.DELIVERY)
 
+    @property
+    def good_standing(self):
+        """Return true if in good standing, otherwise false."""
+        last_ar = self.founding_date
```

Comment:
Is the founding date the anniversary date?

Reply:
That's just the default if they haven't filed a report yet. Got that from the calculation used for anniversary.

Reply:
can you rename the variable and add a comment on how it's calculated or where you're getting the value from? I find last_ar and last_ar_date confusing... we are talking about anniversary date and date of last annual report :)

Reply:
I changed the variable name and added comments